puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/plugins/foreman-tasks.yaml

```diff
@@ -1,8 +0,0 @@
-- project:
```

Comment:
This one runs after merging something. Is that something you wish to drop? The Github Actions don't test that.